src/Sarif.FunctionalTests/SpecExamples/Comprehensive.sarif

```diff
@@ -22,9 +22,11 @@
       },
       "invocation": {
         "commandLine": "CodeScanner @collections.rsp",
-        "responseFiles": {
-          "collections.rsp": "-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999"
-        },
+        "responseFiles": [
+          {
+            "uri": "collections.rsp"
+          }
+        ],
```

Comment:
It doesn't look like you added the response file to the `run.files` dictionary, so now the response file contents don't appear in the log file. Please see the sample JSON I sent in email.

Reply:
So what I have right now is that file.contents is a string. What you gave me was an object, so I figured it was a change we hadn't gotten to yet.

Reply:
Sorry, I can't keep straight in my head the exact order of changes in the [Schema changes](https://github.com/oasis-tcs/sarif-spec/blob/master/Documents/Schema%20changes%20between%20SARIF%20v1%20and%20v2.md) document :-)

So the JSON should be:
```
{                                                  # A run object.
  "files": {                                       # run.files is a dictionary whose property values are file objects.
    "collections.rsp": {                           # A file object
      "contents": "-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999".
    }
  },

  "invocation": {
    "commandLine": "CodeScanner @collections.rsp",
    "responseFiles": [                             # An array of fileLocation objects.
      {                                            # A fileLocation object.
        "uri": "collections.rsp"
      }
    ]
  }
}
```